public/js/fxa-analytics.js

```diff
@@ -155,6 +155,7 @@ function setGAListeners(){
       });
     });
 
+    // Growth Experiment
     if (document.body.dataset.experiment) {
```

Comment:
Unreachable Code – This is gated on the ENV variable, `EXPERIMENT_ACTIVE` set to 1.